Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1348,10 +1347,18 @@ def create_search_with_retry(client: Client, fetch_mode: str, offense: Dict, eve
         f'SELECT {event_columns} FROM events WHERE INOFFENSE({offense_id}) {additional_where} limit {events_limit} '
         f'START {offense_start_time}'
     )
+    print_debug_msg(f'Trying to get events for offense ID: {offense_id}, '
+                    f'query_expression: {query_expression}.')
```

Comment:
Note that the `query_expression` contains a lot of data under `{events_columns}` while staying very static. I think a better approach is to print a truncated version with just the following data:
1. `{offense_start_time}` to show the only dynamic field.
2. `{additional_where}` to show the user query that's being applied.

Reply:
Accepted. Changed to: offense_id, offense_start_time, additional_where, events_limit.